### PR TITLE
fixed digits.py sample to work with opencv 3

### PR DESCRIPTION
**Pull request by techfort**
digits.py python sample contained opencv 2 code, refactored to work with 3.0.0-rc1 API

### CODE REVIEW

File: samples/python2/digits.py [hunk 1]
```diff
@@ -74,30 +74,35 @@ def save(self, fn):
...
     def predict(self, samples):
-        retval, results, neigh_resp, dists = self.model.find_nearest(samples, self.k)
+        retval, results, neigh_resp, dists = self.model.findNearest(samples, self.k)
         return results.ravel()
 
 class SVM(StatModel):
     def __init__(self, C = 1, gamma = 0.5):
-        self.params = dict( kernel_type = cv2.SVM_RBF,
-                            svm_type = cv2.SVM_C_SVC,
+        self.params = dict( kernel_type = cv2.ml.SVM_RBF,
```
Comment (berak) [inline samples/python2/digits.py]: the params struct is gone in 3.0. use `self.model.setKernel(cv2.ml.RBF)` ,  `self.model.setType(cv2.SVM_C_SVC)` , `self.model.setGamma(0.5)` and `self.model.setC(1)`

this also should clear the confusion in line 102 (only the version in line 100 is valid)

Comment (techfort) [inline samples/python2/digits.py]: Cheers @berak - i actually realised that after I posted the pull-request. Want me to correct that and apply parameters through the separate methods?

**Comment by techfort** [comment]
changed the setting of SVM params and used the correct predict signature

**Comment by vpisarev** [comment]
thanks! :+1:

**Pull request merged**